android/gradle.properties

```diff
@@ -2,4 +2,4 @@ customerio.reactnative.kotlinVersion=1.7.21
 customerio.reactnative.compileSdkVersion=33
 customerio.reactnative.targetSdkVersion=33
 customerio.reactnative.minSdkVersion=21
-customerio.reactnative.cioSDKVersionAndroid=4.1.0
+customerio.reactnative.cioSDKVersionAndroid=feature-react-native-cdp-SNAPSHOT
```

Comment:
We'll need to changes this before final release once Android release with changes is live, but the change will go in final PR for CDP

Reply:
lets add a TODO on it

Reply:
lets add a TODO on it

Reply:
Added